libs/cart/src/models/composite-cart-item.ts

```diff
@@ -0,0 +1,12 @@
+import { DaffCartItem } from './cart-item';
+import { DaffCartItemInputType } from './cart-item-input';
+
+export interface DaffCompositeCartItem extends DaffCartItem {
+	type: DaffCartItemInputType.Composite;
+	options: DaffCompositeCartItemOption[];
+}
+
+export interface DaffCompositeCartItemOption {
+	option_label: string;
```

Comment:
Why are these fields suffixed with `_label`?

Reply:
On the composite product, there is an option id, value, and label. It was just for clarity

Reply:
okay so all Daffodil cares about for the cart item is the label, not the actual value?

Reply:
Nothing else comes back from magento anyway, but I think we should just need the labels for the cart item, since you shouldn't be able to edit the item options in the cart.

Reply:
okay that makes sense.